docs/csharp/roslyn-sdk/get-started/syntax-analysis.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Get Started with Syntax Analysis (Roslyn APIs)
+description: An introduction to traversing, querying and walking syntax trees.
+keywords: roslyn, analyzer, code fix
+author: billwagner
+ms.author: wiwagn
+ms.date: 01/29/2018
+ms.topic: conceptual
+ms.prod: .net
+ms.devlang: devlang-csharp
+ms.custom: mvc
```

Comment:
my understanding of the mvc template is that some of the steps would have their own headers. @JRAlexander is probably more familiar with that.